net/Directory.Packages.props

```diff
@@ -1,19 +1,21 @@
 <Project>
-
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
   </PropertyGroup>
-
   <ItemGroup>
+    <PackageVersion Include="coverlet.collector" Version="6.0.2" />
     <PackageVersion Include="Ensure.That" Version="10.1.0" />
+    <PackageVersion Include="FluentAssertions" Version="6.12.1" />
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.2" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
     <PackageVersion Include="PolySharp" Version="1.14.1" />
     <PackageVersion Include="Roslynator.Analyzers" Version="4.12.9" />
     <PackageVersion Include="Roslynator.Formatting.Analyzers" Version="4.12.9" />
     <PackageVersion Include="Substrate.NET.API" Version="0.9.24-rc6" />
     <PackageVersion Include="System.Linq.Async" Version="6.0.1" />
     <PackageVersion Include="System.Threading.Channels" Version="8.0.0" />
+    <PackageVersion Include="xunit" Version="2.9.2" />
+    <PackageVersion Include="xunit.runner.visualstudio" Version="2.8.2" />
```

Comment:
Better group deps for logic and for tests 
https://github.com/gear-tech/sails/blob/5a1bf008eaaba461b4e9ecc168d0d146831331cb/net/Directory.Packages.props